eternalcore-plugin/src/testPaper/java/com/eternalcode/core/IntegrationBaseTest.java

```diff
@@ -0,0 +1,35 @@
+package com.eternalcode.core;
+
+import dev.rollczi.litegration.Litegration;
+import dev.rollczi.litegration.client.Client;
+import dev.rollczi.litegration.client.McProtocolLibClient;
+import java.util.Objects;
+import java.util.function.Consumer;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.Plugin;
+import org.bukkit.plugin.java.JavaPlugin;
+import org.bukkit.scheduler.BukkitScheduler;
+
+public class IntegrationBaseTest {
+
+    private final static Plugin plugin = JavaPlugin.getProvidingPlugin(EternalCoreApi.class);
+    private final static BukkitScheduler scheduler = Bukkit.getScheduler();
+
+
+    protected static void joinPlayer(String name, Consumer<Player> action) {
+        Litegration litegration = Litegration.getCurrent();
```

Comment:
The `getCurrent` could be changed imo on Litegration's end, `"current"` in english without any context means just electricity